nixos/doc/manual/configuration/x-windows.xml

```diff
@@ -96,6 +96,44 @@
   The options are named identically for all other display managers.
   </para>
  </simplesect>
+ <simplesect xml:id="sec-x11--graphics-cards-intel">
+  <title>Intel graphics drivers</title>
+  <para>
+   There are two choices for Intel graphics driver in X.org:
+   <literal>intel</literal> (provided by the package <package>xf86-video-intel</package>)
+   and <literal>modesetting</literal> (included in the <package>xorg-server</package> itself).
+   The default is <literal>modesetting</literal>.
+  </para>
+  <para>
+   The first driver is specific to Intel GPUs but is not recommended by most distribution:
+   it lacks several modern features (Glamor, 2D graphics acceleration via OpenGL, is not
+   possible with the latest DRI 3). The package was also last updated in 2015.
```

Comment:
```suggestion
   compatible with the latest DRI 3). The package was also last updated in 2015.
```

Not sure about that, the driver doesn't support DRI3? I'd assume OpenGL is supported with DRI 3 ;-)

Reply:
This is a point I'm not 100% sure because I've found contrasting information: I came at this conclusion but it may be incorrect, or even nonsense.

xf86-video-intel was the first driver to implement Glamor, back in 2011 (when DRI 3 didn't exist yet). source: [phoronix article](https://www.phoronix.com/scan.php?page=news_item&px=MTAxNDI)

This [page](https://www.freedesktop.org/wiki/Software/Glamor/) mentions the Intel DDX supports Glamor, but I assume it's not updated and talking about DRI 2.

In an arch linux forum [thread](https://bbs.archlinux.org/viewtopic.php?pid=1630467#p1630467) from 2016 I read: 
> modesetting uses glamor for & dri 3 for HW acceleration
> xf86-video-intel uses dri2 and SNA or UXA for HW acceleration.

Which seem to imply it doesn't support Glamor at all.

The Arch wiki article on Intel graphics says
> DRI3 is the default DRI version in xf86-video-intel.

but Glamor is not mentioned anywhere.

Finally, the repository of xf86-video-intel doesn't contain the word "glamor" anywhere and the [manual page](https://gitlab.freedesktop.org/xorg/driver/xf86-video-intel/-/blob/master/man/intel.man) only mentions SNA and UXA methods.

Reply:
Then we shouldn't try to document what's undocumented - let's just state it's quite old and might not be up to date in terms of features, compared to the `modesetting` one :+1: